Training/Training/BoardGame.c

```diff
@@ -0,0 +1,72 @@
+// ------------------------------------------------------------------------------------------------
+// Training ~ A training program for new joiners at Metamation, Batch - July 2024.
+// Copyright Gokul TS (c) Metamation India.
+// ------------------------------------------------------------------
+// BoardGame.c
+// Program to play board game between two players.
+// ------------------------------------------------------------------------------------------------
+#include <stdio.h>
+#include <conio.h>
+#include <stdlib.h>
+
+void PrintBoard (char pieces[3][3]) {
+   printf ("BOARD GAME\n+---+---+---+\n|");
+   for (int i = 0; i < 3; i++)
+      for (int j = 0; j < 3; j++) {
+         printf (j == 0 && i == 1 || j == 0 && i == 2 ? "| %c |" : " %c |", pieces[i][j]);
```

Comment:
A nice feature would be to show each player's pieces (i.e. O or X) in a different colour.